src/styles/main.scss

```diff
@@ -0,0 +1,87 @@
+$main-color: #eee;
+$hover-color: #ffbfcb;
+$font-family: Arial, Halvetica, san-serif;
+$week-days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+$columns: 7;
+$calendar-size: 100px;
+$number: 31;
+$days-gap: 1px;
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  width: ($calendar-size * $columns) + (6 * $days-gap);
```

Comment:
The use of arithmetic operations in CSS should be avoided where possible for readability and maintainability. Consider calculating the width in a separate variable with a descriptive name, and then use that variable for the width property.